src/features/registry/middleware.ts

```diff
@@ -23,6 +23,10 @@ export const basicApiKeyAuthenticate: RequestHandler = async (
 		req.params.subject = creds.name;
 		req.params.apikey = creds.pass;
 	}
+	if (req.params.apikey === TOKEN_AUTH_BUILDER_TOKEN) {
```

Comment:
Is the `TOKEN_AUTH_BUILDER_TOKEN` actually used any more?
I couldn't find any references outside the API :thinking:
See: https://github.com/search?type=code&auto_enroll=true&q=%22TOKEN_AUTH_BUILDER_TOKEN%22+-path%3A*.md+-path%3A%2F.versionbot%2F*+NOT+is%3Aarchived+%28org%3Abalena-io+OR+org%3Abalena-io-examples+OR+org%3Abalena-io-library+OR+org%3Abalena-io-modules+OR+org%3Abalena-os+OR+org%3Abalenablocks+OR+org%3Abalena-labs-projects+OR+org%3Abalenaltd+OR+org%3Aproduct-os%29

Reply:
It's certainly set, I don't remember what uses it but I'm also not fussed about it in terms of this PR as it's definitely not something to bundle with it